frontend/public/components/persistent-volume.jsx

```diff
@@ -8,7 +8,12 @@ import { Kebab, LabelList, navFactory, ResourceKebab, SectionHeading, ResourceLi
 const { common } = Kebab.factory;
 const menuActions = [...common];
 
-const PVStatus = ({pv}) => <StatusIconAndText status={pv.status.phase} />;
+export const pvPhase = pv => pv.status.phase;
+
+const PVStatus = ({ pv }) => {
+  const phase = pvPhase(pv);
+  return <StatusIconAndText status={phase} />;
```

Comment:
FYI #1697 changes this.

Reply:
Removing these changes for now from this PR.